src/map.rs

```diff
@@ -2168,12 +2168,568 @@ fn assert_covariance() {
     }
 }
 
+impl<K, V, S> HashMap<K, V, S> {
```

Comment:
Could you move this (and the other) impl blocks so that they line up with where the corresponding impl blocks are in hashbrown's `src/map.rs`? That'll make it much easier to keep the codebase up to date over time, since we can just diff the files directly!

Reply:
Yup, no problem.

Reply:
Done. :+1: